src/app/components/ActiveUserLoanContainer/components/ActiveLoanTableContainer/index.tsx

```diff
@@ -146,9 +147,13 @@ export function ActiveLoanTableContainer(props: Props) {
     }
   }, [expandedId]);
 
+  useEffect(() => {
+    window.addEventListener('resize', () => setWindowSize(window.innerWidth));
+  }, []);
+
   return (
     <>
-      {window.innerWidth > 991 && (
+      {windowSize > 991 && (
```

Comment:
I think it's better to use css media queries to hide / show elements, you can do it by wrapping component with div

```suggestion
      <div className="d-none d-md-block">
```